generators/server/templates/_build.gradle

```diff
@@ -27,8 +27,12 @@ apply plugin: 'maven'
 apply plugin: 'spring-boot'
 apply plugin: 'war'
 apply plugin: 'propdeps'
-<%_ if (applicationType === 'microservice' || applicationType === 'gateway' || applicationType === 'uaa') { _%>
 apply plugin: 'io.spring.dependency-management'
+<%_ if (databaseType == 'sql') { _%>
+/* downgrade Hibernate to 4.3 */
+ext['hibernate.version'] = '${hibernate_entitymanager_version}'
```

Comment:
This is the recommended way of downgrading to Hibernate 4.3 from Spring Boot.

https://github.com/spring-projects/spring-boot/issues/6507